### ISSUE TITLE
Actual called times test descriptions

### ISSUE THREAD

**Issue by gwigz** [kahlan/kahlan#239]
Trying to debug this, I know it's related to `Calls::find` and maybe the logger, as the adjustments I made resolved some issues with the tests.

I've made a [hacky attempt here](https://github.com/kahlan/kahlan/compare/dev...gwigz:6ce1906) that resolved part of the issue explained below.

Before the changes, if you called a method 3 times or more, but expected it to be called twice, the description would look like this:

```
actual:
  (string) "time()"
actual called times:
  (integer) 2
expected to be called:
  (string) "time()"
expected called times:
  (integer) 2
```

The part I can't figure out, after my changes this now happens:

- Expect a function to be called 2 times
    - Call it 1, 2 or 3 times, the description is correct!
    - Call it more than 3 times, the description says it was called 3 times, rather than maybe 3+ times maybe?

It's definitely related to the logic in `Calls::find` on the `$times` parameter, but how to go about changing how that's handled, but this is already beyond my understanding of Kahlan at the moment.

**Comment by jails** [comment]
Ah good catch :+1: , you meant [something like that](https://github.com/kahlan/kahlan/commit/c271cede156e6e59c309a4fa991f648faeb69afd) ?

**Comment by gwigz** [comment]
Perfect! 😋

**Comment by jails** [comment]
Cool, merged in 3.1.14.
